content/soarian/r4/base/entities/device.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Device | R4 API
+---
+
+# Device
+
+* TOC
+{:toc}
+
+## Overview
+
+The Device resource Implementation is currently limited to devices such as pacemakers that are implanted in a patient as part of a charted procedure. As such, this resource currently exposes the known implant history for a patient occurring at this site. 
+
+_Soarian Clinicals_<sup>®</sup> supports a read-only Application Programming Interface (API). This API accepts `GET` and `POST` based [search] and `GET` based [read] interactions. The response represents the most current information about the patient that is charted in _Soarian Clinicals_ at the time of the query. 
```

Comment:
In the second sentence, remove the italics from the Soarian Clinicals reference and add a registered trademark. It should look the same as the reference to Soarian Clinicals in the first sentence.

Reply:
Thanks for the review, I have removed italics font and added trademark to Soarian Clinicals.